vertical-pod-autoscaler/pkg/utils/vpablessing/vpa_blessing.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vpablessing
```

Comment:
As much as I like the VPA blessing name (hoping VPA is a blessing and not a curse ;) ) I'd prefer something more descriptive. observedContainers maybe?

Reply:
I'm also wondering how to structure this code. My intuition is that we could put code that deals with annotations in one place so we would have
utils/annotations/updates.go
utils/annotations/observed_containers.go
but I expect there might be more code connected with vpaBlessing that updater will need. LMK how you expect this to be structured.

Reply:
I thought of smth like this:
1. In `utils` we have common code for parsing and creating annotation value. There is also an annotation name const.
2. Updating code will be in the Admission Controller. This is just applying patch. This type of code is already there. Label name and value will be provided by util consts/functions.
3. The updater will read annotation. There should be some util function for retrieving annotation with given name. Parsing will be done by annotation util. The only thing that requires more than 2 lines should be `if` for omitting not listed components.

> I'd prefer something more descriptive. observedContainers maybe?

I don't have a strong opinion here. I can go with "observedContainers". "VpaBlessing" is definitely more memorable, though...

Reply:
After the code update, current naming is:
- `annotations` - package
- `vpaObservedConstainers` instead of `vpaBlessing`
- `vpa_observed_containers.go` instead of `vpa_blessing.go`